gapii/cc/spy_base.h

```diff
@@ -185,6 +185,8 @@ class SpyBase {
   track_memory::MemoryTracker mMemoryTracker;
 #endif  // TARGET_OS
 
+  // This is true if we should not show disabled extensions to the application.
```

Comment:
nit: this comment was hard for me to parse :) I think it be better phrased as "If true, we will hide disabled extensions from the application."

Reply:
Sure